tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,375 @@
+"""Tests for leapp upgrade of content hosts with Satellite
+
+:Requirement: leapp
+
+:CaseLevel: Integration
+
+:CaseComponent: LeappIntegration
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from broker import Broker
+
+from robottelo.cli.activationkey import ActivationKey
+from robottelo.cli.repository import Repository
```

Comment:
Please remove these imports and use `module_target_sat.cli.ActivationKey`  and `module_target_sat.cli.Repository` instead.